sdks/wasm/DebuggerTestSuite/DateTimeTests.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+using System.Net.WebSockets;
+using System.Threading;
+using System.IO;
+using System.Text;
+using System.Collections.Generic;
+
+using Newtonsoft.Json.Linq;
+using Xunit;
+using WebAssembly.Net.Debugging;
+using System.Globalization;
+
+namespace DebuggerTests
+{
+	public class DateTimeList : DebuggerTestBase {
+		public DateTimeList() : base ("debugger-driver.html") {}
+
+		[Fact]
+		public async Task CheckThatAllLocaleSourcesAreSent () {
+			var insp = new Inspector ();
+			//Collect events
+			var scripts = SubscribeToScripts(insp);
+
+			await Ready();
+			//all sources are sent before runtime ready is sent, nothing to check
+			await insp.Ready ();
+			Assert.Contains ("dotnet://debugger-test.dll/debugger-datetime-test.cs", scripts.Values);
+		}
+
+		[Theory]
+		[InlineData ("en-US")]
+		// [InlineData ("es-ES")]
+		[InlineData ("de-DE")]
+		// [InlineData ("ja-JP")]
+		[InlineData ("ka-GE")]
+		[InlineData ("hu-HU")]
+		public async Task CheckDateTimeLocale (string locale) {
+			var insp = new Inspector ();
+			var scripts = SubscribeToScripts(insp);
+
+			await Ready();
+			await insp.Ready (async (cli, token) => {
+				ctx = new DebugTestContext (cli, insp, token, scripts);
+				var debugger_test_loc = "dotnet://debugger-test.dll/debugger-datetime-test.cs";
+
+				await SetBreakpoint (debugger_test_loc, 20, 3);
+				
+				var pause_location = await EvaluateAndCheck (
+					"window.setTimeout(function() { invoke_static_method ('[debugger-test] DebuggerTests.DateTimeTest:LocaleTest'," 
+					+ $"'{locale}'); }}, 1);",
+					debugger_test_loc, 20, 3, "LocaleTest",
+					locals_fn: (locals) => {
+						CultureInfo.CurrentCulture = new CultureInfo (locale, false);
```

Comment:
Maybe do this after `CultureInfo.GetCultureInfo(locale).DateTimeFormat;`, because this can affect the result of that second one. Documentation says that it would cause it to not reflect "user overrides", I'm not sure what that means, but might be safer to do that?